src/styles/searchResults.css

```diff
@@ -0,0 +1,42 @@
+.search-results {
```

Comment:
Should make sure you use the same level of indentation throughout, so this should be brought in to 2 spaces as with everywhere else
Also always make sure that the closing brace of the class is at the same level of indentation as the opening one